### PR TITLE
Build System: Add Flatpak CI

### PR DESCRIPTION
**Pull request by gal20**
Based on #3770, I created a pipeline with an updated flatpak manifest that successfully builds the application. The Builds are found [here](https://dev.azure.com/gal20/gal20.yuzu/_build).

To set up the pipeline, you'll need to upload your private ssh and gpg keys (named 'ssh.key' and 'gpg.key') as secure files, and create a variable group called 'flatpak-variables' that contains the flatpak environment variables.

Changed required before merging: 
- Uncomment the sshfs command
- Change the docker image to yuzu's (currently uses meirod's, because the linux-flatpak image doesn't exist in yuzuemu/build-environment)
- Update the runtime version to 5.15  (see yuzu-emu/build-environments#21)

Update 27.1:
The builds now fail to compile yuzu because the 5.13 runtime is too old. On the 5.15 runtime it compiles successfully.

### CODE REVIEW

File: .ci/scripts/linux-flatpak/finish.sh [hunk 1]
```diff
@@ -5,5 +5,5 @@ REPO_DIR="$YUZU_SRC_DIR/repo"
 
 # When the script finishes, unmount the repository and delete sensitive files,
 # regardless of whether the build passes or fails
-#umount "$REPO_DIR" || true
-#rm -rf "$REPO_DIR" "/tmp/*" || true
+umount "$REPO_DIR" || true
+rm -rf "$REPO_DIR" "/tmp/*" || true
```
Comment (v1993) [inline .ci/scripts/linux-flatpak/finish.sh:9]: Change this to

```bash
rm -r "$REPO_DIR" /tmp/* || true
```

It's no use to hide errors from log and quoting `/tmp/*` makes it fail. Try this out locally to see:

```bash
mkdir test
touch test/{1..3}
rm -r "test/*"
```

Result is `rm: cannot remove 'test/*': No such file or directory`.

**Review by v1993 — CHANGES_REQUESTED** [review]

**Review by v1993 — CHANGES_REQUESTED** [review]
A slight issue there

**Comment by soredake** [comment]
https://github.com/flathub/org.citra_emu.citra

**Comment by soredake** [comment]
https://flathub.org/apps/details/org.yuzu_emu.yuzu

**Pull request closed without merging**